ui/components/src/GaugeChart/GaugeChart.tsx

```diff
@@ -128,7 +140,7 @@ export function GaugeChart(props: GaugeChartProps) {
             show: false,
           },
           detail: {
-            show: true,
+            show: width > GAUGE_SMALL_BREAKPOINT, // hide center value at narrow chart widths
```

Comment:
Thoughts for follow-up work (i.e. not expecting to fix in this PR): I wonder if we should have a tooltip or something for these cases, so it's still possible to get the value.